app/actions/departments/department.go

```diff
@@ -0,0 +1,124 @@
+package departments
+
+import (
+	"mjm/app/models"
+	"mjm/app/render"
+	"net/http"
+
+	"github.com/gobuffalo/buffalo"
+	"github.com/gobuffalo/pop/v6"
+)
+
+var (
+	r = render.Engine
+)
+
+func List(c buffalo.Context) error {
+	tx := c.Value("tx").(*pop.Connection)
+
+	departments := []models.Department{}
+	err := tx.All(&departments)
+	if err != nil {
+		return err
+	}
+
+	c.Set("department", departments)
```

Comment:
Please rename the key from "department" to "departments", that represents a collection of departments.